R/fit.R

```diff
@@ -36,7 +36,7 @@ epidist <- function(data, formula, family, prior, backend, fn, ...) {
 #' @method epidist default
 #' @family fit
 #' @export
-epidist.default <- function(data, formula = brms::bf(mu ~ 1),
+epidist.default <- function(data, formula = mu ~ 1,
```

Comment:
documentation needs an update

Reply:
Documentation updated in https://github.com/epinowcast/epidist/pull/397: sorry